templates/docker-monolithic/restore.sh

```diff
@@ -41,8 +41,10 @@ if echo "${TAR_OUTPUT}" | grep -qs '^volumes-monolithic/$'; then
 
   docker exec supportpal bash -c "cp -r ${TEMP_BACKUP_DIR}/volumes-monolithic/cache/redis-data /"
   docker exec supportpal bash -c "cp -r ${TEMP_BACKUP_DIR}/volumes-monolithic/caddy/caddy /"
+  docker exec supportpal bash -c "cp -r ${TEMP_BACKUP_DIR}/volumes-monolithic/meilisearch /"
```

Comment:
```suggestion
  if echo "${TAR_OUTPUT}" | grep -qs '^volumes-monolithic/meilisearch'; then
    docker exec supportpal bash -c "cp -r ${TEMP_BACKUP_DIR}/volumes-monolithic/meilisearch /"
  fi  
```

This needs wrapping otherwise it produces `cannot stat` error when restoring an old backup (which doesn't contain meilisearch volume data)